### ISSUE TITLE
ARVideoKit License 

### ISSUE THREAD

**Issue by Jlack6** [AFathi/ARVideoKit#116]
I recently added ARVideoKit to my application ScribScrab. I want to start this off by saying the kit is incredible and the documentation is very easy to use. I'm going to try and release an update of ScribScrab using the ARVideoKit on the App Store soon but I wanted to reach out about the licence and how I go about it? I wanted to make sure that I give ARVideoKit the proper credit. Also would it be possible to get ScribScrab on the "Apps using ARVideoKit" list?

**Comment by AFathi** [comment]
Hi @Jlack6, please add the license in the settings section of your application under a section called `Licenses` where you'd list the open source code you've used throughout the app. If you don't have a settings section in the app, you may add the license in the Settings bundle (where it can be viewed from the System Settings); see instructions to do so here: https://gist.github.com/zetachang/4111314

Of course, please fork the repository, add ScribScrab to the README file and create a pull request!

**Comment by Jlack6** [comment]
@AFathi Hey, thanks for getting back to me so quickly. I don't have a settings section in the application but I have a link on the home screen that goes to a website I built for the app (The website is still a work in progress). I built this link on the site for acknowledgements https://scribscrabweb.firebaseapp.com/Acknowledgements.html, does that work? Let me know if I need to change anything. I'll also try to make sure I get that Settings bundle in soon. I'm kind of new to this so sorry for all the questions and thanks for the help.
